lib/account/pages/reviews_page.dart

```diff
@@ -13,18 +13,15 @@ import '../widgets/review_detail.dart';
 import 'write_review_page.dart';
 
 class ReviewsPage extends StatefulWidget {
-  final int profileId;
-  final Profile? profile;
+  final Profile profile;
 
-  const ReviewsPage({super.key, required this.profileId}) : profile = null;
```

Comment:
why did you remove the constructor here? we left it everywhere else, and it might prove useful for notification deeplinks in the future

Reply:
mhh ok. I removed it because the const with the id does not work at the moment. Or at least every test I tried for it failed. I will take a look at it again and add it back in (worse case its not tested and will be needed to be added once its needed)

Reply:
? don't we have dozens of other pages where this works?

Reply:
The constructor it self works but not the build since the fullyloaded is not used. But ill take care of it

Reply:
It's fine, we can add it back when/if we need it